src/main/java/net/earthcomputer/clientcommands/command/CTickCommand.java

```diff
@@ -0,0 +1,210 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import net.earthcomputer.clientcommands.event.MoreClientEvents;
+import net.earthcomputer.clientcommands.task.SimpleTask;
+import net.earthcomputer.clientcommands.task.TaskManager;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents;
+import net.minecraft.client.Minecraft;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.protocol.game.ClientboundSetTimePacket;
+import net.minecraft.util.TimeUtil;
+
+import java.text.DecimalFormat;
+
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.*;
+
+public class CTickCommand {
+
+    private static final DecimalFormat DEC_FMT = new DecimalFormat("0.00");
+
+    private static final String TASK_NAME = "ctick";
+
+    private static TickMeasuringTask currentMeasurer = null;
+
+    public static void register(CommandDispatcher<FabricClientCommandSource> dispatcher) {
+        dispatcher.register(literal("ctick")
+            .then(literal("client")
+                .then(literal("tps")
+                    .executes(ctx -> getClientTps(ctx.getSource())))
+                .then(literal("mspt")
+                    .executes(ctx -> getClientMspt(ctx.getSource()))))
+            .then(literal("server")
+                .then(literal("tps")
+                    .executes(ctx -> getServerTps(ctx.getSource())))
+                .then(literal("mspt")
+                    .executes(ctx -> getServerMspt(ctx.getSource())))));
+    }
+
+    private static int getClientTps(FabricClientCommandSource source) throws CommandSyntaxException {
+        stopPreviousTask();
+
+        TickMeasuringTask measurer = new TickMeasuringTask(true, false);
+        TaskManager.addTask(TASK_NAME, measurer);
+        currentMeasurer = measurer;
+
+        float tps = source.getWorld().tickRateManager().tickrate();
+        source.sendFeedback(Component.translatable("commands.ctick.client.tps.expectedTps", tps));
```

Comment:
These expected TPS and MSPT messages shouldn't be printed while the game is sprinting